File: src/main/java/com/hanocybous/model/SimpleTask.java
package com.hanocybous;

public final class SimpleTask extends Task {

    public SimpleTask(int id, String name, int mamaId, int start, int end, double d) {
        super(id,name,mamaId,start,end,d);
    }

    public void addSubTask(Task subTask) { /*Simple Tasks can't have subtasks*/ }
}

File: src/main/java/com/hanocybous/model/SimpleTask.kt
package com.hanocybous.model

class SimpleTask(
    id: Int,
    name: String,
    mamaId: Int,
    start: Int,
    end: Int,
    cost: Double
) : Task(id, name, mamaId, start, end, cost) {

    override fun addSubTask(subTask: Task) {
        // Simple Tasks can't have subtasks
    }
}